test/jdk/java/lang/CompressExpandSanityTest.java

```diff
@@ -22,10 +22,11 @@
  */
 
 /*
- * @test
- * @summary Test compress expand as if the test methods are the implementation methods
- * @key randomness
- * @run testng CompressExpandSanityTest
+ * //@test
+ * //@summary Test compress expand as if the test methods are the implementation methods
+ * //@key randomness
+ * //@run testng/othervm -XX:+UnlockDiagnosticVMOptions -XX:DisableIntrinsic=_expand_i,_expand_l,_compress_i,_compress_l CompressExpandSanityTest
+ * //@run testng CompressExpandSanityTest
```

Comment:
```suggestion
// Disabled by default
// @test
/*
 * @summary Test compress expand as if the test methods are the implementation methods
 * @key randomness
 * @run testng/othervm -XX:+UnlockDiagnosticVMOptions -XX:DisableIntrinsic=_expand_i,_expand_l,_compress_i,_compress_l CompressExpandSanityTest
 * @run testng CompressExpandSanityTest
 */
```
I think we need to do it like above, otherwise it will induce a test error